specification/electronic-prescription-service-api.yaml

```diff
@@ -812,6 +837,7 @@ paths:
             application/fhir+json:
               schema:
                 type: object
+                description: TBC
```

Comment:
do we need to address the TBC now?

Reply:
Good spot - I've removed that line for now.